src/pal/inc/pal.h

```diff
@@ -287,6 +274,19 @@ PAL_IsDebuggerPresent(VOID);
 #define LONG_MAX      2147483647L
 #define ULONG_MAX     0xffffffffUL
 
+#endif // !PAL_STDCPP_COMPAT
+
+/******************* PAL-Specific Entrypoints *****************************/
+
+#define IsDebuggerPresent PAL_IsDebuggerPresent
+
+PALIMPORT
+BOOL
+PALAPI
+PAL_IsDebuggerPresent(VOID);
+
+#define MAXIMUM_SUSPEND_COUNT  MAXCHAR
```

Comment:
I do not see `MAXIMUM_SUSPEND_COUNT` used anywhere. It would be better to delete it.

Reply:
Will remove. The compilation succeeded with higher version of clang locally on Ubuntu, but clang 3.9 is failing in CI. I will install it. (would be cool if Unix toolset also get updated as frequently as MSVC https://github.com/dotnet/coreclr/issues/17684).